src/app/app-routing.module.ts

```diff
@@ -1,7 +1,13 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 
-const routes: Routes = [];
+import { CoursesPageComponent } from './pages/courses-page/courses-page.component';
+
+const routes: Routes = [
+  { path: 'courses', component: CoursesPageComponent },
+  { path: '', redirectTo: 'courses', pathMatch: 'full' },
```

Comment:
courses and similar string which belongs to app routing might be moved to constants or enum (e.g. `AppRoutePath`). this will allow you to not repeat yourself as well as give you other benefits.

Reply:
Will create separate enum